gnuradio-runtime/include/gnuradio/rpcmanager.h

```diff
@@ -53,7 +53,7 @@ class GR_RUNTIME_API rpcmanager : public virtual rpcmanager_base
   static bool make_aggregator;
   static bool booter_registered;
   static bool aggregator_registered;
-  static void rpcserver_booter_base_sptr_dest(rpcserver_booter_base* b) {;}
+  static void rpcserver_booter_base_sptr_dest(rpcserver_booter_base* b) {(void) b;}
```

Comment:
I swing towards void casting to avoid the unused problem.

Motivation: `__attribute__((unused))` is far too long and also, not standard C++.

going the route of having a configure.h that defines a `GR_UNUSED` (identical to what UHD_UNUSED is) to deal with compilers that don't support this attribute setting is just another code difference we add to suppress warnings on a subset of compilers. 

So, I'd prefer `(void) b; // Suppress the "unused variable" compiler warning` thing at the very top of affected functions, and if applicable, doxygen `\param b (unused)`.

Reply:
To be clear, I was not advocating for `GR_UNUSED()`, I was advocating not naming the variable in the function signature.